lib/shp-reader.js

```diff
@@ -82,7 +90,14 @@ ShpReader.prototype.readRecord = function (callback) {
       }
 
       // shapeType is good, parse contentBuffer
-      const content = new shapeTypeParsers[SHAPE_TYPES[shapeType]](contentBuffer)
+      let content
+      if (shapeTypeParsers.hasOwnProperty(SHAPE_TYPES[shapeType])) {
+        content = new shapeTypeParsers[SHAPE_TYPES[shapeType]](contentBuffer)
+      } else {
```

Comment:
Sure, but the `dagi_kommune` has some shape types there aren't fully supported. But it also tests some things in the `.bdf` parser (large files and wired trailer). So I can't just let it fail. But once we got proper support for `dagi_kommune` that should definitely be changed to emit en error.